README.md

```diff
@@ -1,24 +1,80 @@
-# Rust binding to [Open Quantum Safe][oqs]'s [liboqs][]
+liboqs-rust: Rust bindings for liboqs
```

Comment:
The naming convention of Rust crates would favour `oqs-rs`, but that's not a hard rule.

Reply:
Also there are two separate crates that would need to be renamed here.

Reply:
Can I call the GitHub repo liboqs-rust but then continue to use your crate names?  Or is there some convention that the repo name also match the crate names?

Reply:
I think that would be fine.